CHANGELOG.md

```diff
@@ -5,6 +5,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Added
+
+- Committed banner for home page
+- Committed page
+
+## Changed
+
+- Podcast_series_pre_get_posts() to look at serial categories array
```

Comment:
Don't need to capitalise this as the function name isn't capitalised makes sense to match it exactly